packages/data-explorer/src/routes.ts

```diff
@@ -1,26 +1,36 @@
 import MainView from './views/MainView.vue'
+import DataRowEdit from '@/components/DataRowEdit.vue'
 
-const routes = [
+export const defaultRouteQuery:any = { page: 1, size: 20, hideSidebar: 'false', view: 'CardView' }
+
+export const routes = [
   {
-    path: '/:entity/:view',
+    path: '/explorer/:entity',
     component: MainView,
-    name: 'main-view'
+    name: 'de-view'
+  },
+  {
+    path: '/explorer/:entity/create',
+    props: true,
+    name: 'de-create',
+    component: DataRowEdit
   },
   {
-    path: '/:entity?',
+    path: '/explorer/:entity/:dataRowId/edit',
+    props: true,
+    name: 'de-edit',
```

Comment:
why the 'de' prefix , suggest to replace with 'edit'

Reply:
Its useful to prefix actions with a module, so that you don't get name collisions when merging other modules (e.g. the navigator or data-row-edit)

Reply:
its not a module but a view ( or should be a view)